### ISSUE TITLE
Suggestions from Ariel Chavez at the CA State Water Board

### ISSUE THREAD

**Issue by spennino** [openoakland/clean-water#5]
I'm sure some of the following may already be on the list for their future improvements, but here are some suggestions I have:

- [ ] Can the graphics be made to be clicked on so they may be viewed larger?
Wendy would probably want an asterisk by "unsafe drinking water" to define how we associate non-compliance with exceeding primary state/federal drinking water standards
- In regards to the links in top left-hand corner
  - [ ] Maybe rename the last link to "Map of Chronic Violations" (so viewers will see the word "Map" first) or "Map of Continuous Violations"
  - [ ] For top right, maybe reword as "Reporting Period : 2012-Current*"
- In regards to Map"
  - [ ] Review colors of "splashes" for color contrast; the grey over the background blue is a little hard to see in some places (Accessibility Issue)
  - [ ] Something I had to do with our HR2W map (and still need to work on) is using language around 6th grade level to ensure most people can understand it. Perhaps instead of the work chronic, use "continuous?" Just one example. Perhaps wording across the site can be reviewed and edited appropriately.
- In regards to Leaderboard:
  - [ ] Capitalize Title, maybe reword? Maybe: "Population at Risk by California State Assembly District"
  - [ ] Wording Example- 198,973 constituents without access to clean drinking water? This is a population count right?
  - [ ] Check color contrast of wording against blue background
  - [ ] Add button to return to home

**Comment by sharkberto** [comment]
@ruckeralex to give you an update, I am looking at how to properly adjust the colors on the interactive map display
